qdlt/qdltargument.cpp

```diff
@@ -391,7 +391,7 @@ QString QDltArgument::toString(bool binary) const
         break;
     case DltTypeInfoStrg:
         if(data.size()) {
-            text += QString("%1").arg(QString(getData()));
+            text += QString::fromLatin1(data.data());
```

Comment:
```suggestion
            text += QString::fromLatin1(data);
```

should work the same way